documentation/monitoring.md

```diff
@@ -75,6 +75,42 @@ There are several other variables that can be changed depending on env requireme
 - grafana_app_mem - app memory limit (default 1Gi)
 - grafana_app_cpu - app requests cpu (default 500m)
 
+Grafana could be accessed via the respective URL's based on the environment of interest. The URL's corresponding to each environment is as below
+
+### Test | https://grafana.test.teacherservices.cloud 
+### Production | https://grafana.teacherservices.cloud/
+
+The default access to the grafana interface is view only, which does not require authentication. In order to be able to make changes for example adding more dashboards and editing existing dashboards, requests will have to be made in the Teacher-services-Infra slack channel to obtain admin credentials 
+
+### Editing or creating new grafana dashboards
+The following steps are required for creating or editing dashboards
+-   Ensure you are logged in as an admin
+-   Identify the purpose of your dashboard. What insights the dashboard will provide and what messages it  conveys
+- 	Plan and Design how the dashboard would look when completed, paying attention to the placement of panels, alignment, spacing, colour 
+    and organisation
+- 	Select the Appropriate Data Sources by identifying the right datasource to visualise in the dashboard (examples of usual datasources 
+    are prometheus, , databases, APIs, log files)
+-	Create Panels for each metric by adding panel for the metric and choosing the right visualisation (for example graph, gauge, table, 
+    heatmap) and configure the panel settings eg the query, data transformation and diaplay options and add concise title for clarity
+
+
+### Exporting a Grafana Dashboard as json
+Grafana allows you to export your dashboard as a JSON file, which can be version controlled and shared with others. This could be achieved by following these steps
+- 	Open your dashboard in Grafana
+-	Click on the "Share" button(icon) in the top left corner
+-	In the "Export" tab, select "Export for sharing externally"
+-	Click "Save to file" to download the JSON file of your dashboard
+
+### Creating a Grafana Dashboard from existing json
+
+- Log in to Grafana as admin
+- Navigate to the Dashboard Import Page and click the "+" icon in the left sidebar to open the dashboard menu, select "Import" from the  
+  dropdown menu to access the dashboard import page.
+- Import the JSON File by either clicking on "Upload JSON file" and selecting the json file from your computer or pasting the json file 
+  content into the text area provided
+-  Click on the "Import" button to initiate the dashboard import process
+
+
```

Comment:
should explain that dashboard additions/updates will be overwritten and need to be PR'd to be permanent. List where/how you would add the dashboard via PR

Reply:
done